lisa/sut_orchestrator/azure/features.py

```diff
@@ -1514,12 +1514,12 @@ def _enable_secure_boot(cls, *args: Any, **kwargs: Any) -> None:
                 "DiskWithVMGuestState" if settings.encrypt_disk else "VMGuestStateOnly"
             )
 
-            virtual_machines["properties"]["storageProfile"]["osDisk"][
+            template["functions"][0]["members"]["getOSImage"]["output"]["value"][
                 "managedDisk"
             ] = (
-                "[if(not(equals(parameters('nodes')[copyIndex('vmCopy')]['disk_type'], "
+                "[if(not(equals(parameters('node')['disk_type'], "
```

Comment:
Is it possible to move this string into arm template?

Reply:
Is it possible for a feature to add arm template parameters? Would it be done in `on_before_deployment` or somewhere else? I would probably need three new parameters to control all the possible CVM configurations.

Reply:
Yes, the arm template parameters are modified in this class too. It's good to control by parameters if it's possible.

Reply:
I will makes these changes in another PR.